development/includes/_top-nav.pug

```diff
@@ -1,13 +1,13 @@
 // Classes are currently placeholder with zero styling. Removed bootstrap
 nav.navbar
```

Comment:
You're code should follow this format: 
```
nav.navbar
  button
    i(class="fa fa-bars fa-2x")

  h1
    a(href="/index.html") MsPudding

  ul
    li
      a(href="/blog/chapter1htmlcss.html") Blog
    li
      a(href="/about.html") About
```
Notice there is not class names on anything inside the navbar, they're not needed because all the element are unique and semantic. Don't wrap you elements with extra div tags for no reason, that's called "divitis" like a disease haha. The presentation is controlled 100% by css, so if something doesn't look right, we'll fix it in the css.